JOSESwift/Sources/Algorithms.swift

```diff
@@ -34,8 +34,10 @@ public enum SignatureAlgorithm: String {
 /// An algorithm for asymmetric encryption and decryption.
 ///
 /// - RSA1_5: [RSAES-PKCS1-v1_5](https://tools.ietf.org/html/rfc7518#section-4.2)
+/// - RSAES-OAEP: [RSAES-OAEP](https://tools.ietf.org/html/rfc7518#section-4.3)
 public enum AsymmetricKeyAlgorithm: String {
     case RSA1_5 = "RSA1_5"
+    case RSAES_OAEP = "RSAES-OAEP"
```

Comment:
A little picky but, I think we should name it like in the JWA rfc, as the underscore in `RSA1_5` is only because the algorithm parameter is named like this.

Reply:
hum... in this case whats your suggestion for the name? The RFC uses RSAES-OAEP or RSAES OAEP, both are not possible in swift enums. RSAESOAEP? just OAEP?

Reply:
Sorry 🤦‍♂️ forgot about that. Looking at the SymmetricKeyAlgorithm, `A256CBC-HS512` is written as `A256CBCHS512`, so `RSAESOAEP` it is.

Reply:
You're right @garrefa 🙂 `-`is not possible in Swift enums.

I think the "alg" [parameter valuehttps://tools.ietf.org/html/rfc7518#section-4.1) for the algorithm is `RSA-OAEP` so what would you think about going with `RSA_OAEP`? Not perfect, I know but probably the closest we can get.

Reply:
Would go for `RSAOAEP` 😁

Reply:
Fixed

Reply:
Hello guys. I think it's amazing that you decided to fill the gap and create this library. It is super useful for newbies in cartography like me. 

I just want to let you know that I did some testing of this branch and it seems that [JwtSecurityTokenHandler](https://msdn.microsoft.com/en-us/library/system.identitymodel.tokens.jwtsecuritytokenhandler(v=vs.114).aspx) class from .NET framework is not able to decode tokens with `"alg": "RSAES-OAEP"` parameter in the header, yet it works fine with `RSA-OAEP` version. I think that it may be connected with @daniel-mohemian's suggestion that according to RFC standard this param should has value `RSA-OAEP`.